src/apps/explorer/components/OrdersTableWidget/useSearchInAnotherNetwork.tsx

```diff
@@ -106,7 +112,7 @@ export const EmptyOrdersMessage = ({
                 {ordersInNetworks.map((e) => (
                   <li key={e.network}>
                     <Link
-                      to={`/${PREFIX_BY_NETWORK_ID.get(e.network)}/address/${ownerAddress}`}
+                      to={_buildInternalNetworkUrl(e.network, ownerAddress)}
```

Comment:
I could have left it as `${PREFIX_BY_NETWORK_ID.get(networkId) && '/' + PREFIX_BY_NETWORK_ID.get(networkId)}/address/${ownerAddress}`

But I think it's already starting to become less readable @alfetopito .